test/e2e/network_test.go

```diff
@@ -0,0 +1,31 @@
+package e2e
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// testNetwork runs all the cluster and node network tests
+func testNetwork(t *testing.T) {
+	t.Run("OpenShift HNS networks", testHNSNetworksCreated)
+}
+
+// testHNSNetworksCreated tests that the required HNS Networks have been created on the bootstrapped nodes
+func testHNSNetworksCreated(t *testing.T) {
```

Comment:
Is this enough or should we also be checking if the hybrid-overlay is running?

Reply:
Sure. I can check that also. Do you have any other requests?